demisto_sdk/commands/content_graph/interface/neo4j/queries/relationships.py

```diff
@@ -32,7 +32,7 @@ def build_target_properties(
     identifier: str = "object_id",
     with_content_type: bool = False,
 ) -> str:
-    properties = {identifier: "rel_data.target"}
+    properties = {identifier: "rel_data.target", "not_in_repository": False}
```

Comment:
The reason for this is because of this scenario:
1. Content Item "a" uses "region" by name, which is not in repository. 
2. We create "region" node
3. Content item "b" uses "region' by id, which is in the repository.
4. Two relationships are created for both nodes.